source/includes/fact-write-blocking-enable.rst

```diff
@@ -1,9 +1,26 @@
-``mongosync`` does not enable write-blocking by default. If you enable
-write-blocking, ``mongosync`` blocks writes:
+By default, ``mongosync`` enables destination-only 
+write-blocking on the destination cluster. 
+``mongosync`` unblocks writes right before the 
+:ref:`/progress <c2c-api-progress>` endpoint reports 
+that ``canWrite`` is ``true``. You can explicitly
+enable destination-only write-blocking by using
+the :ref:`/start <c2c-api-start>` endpoint to set
+``enableUserWriteBlocking`` to ``"destinationOnly"``.
 
-- On the destination cluster during sync.
-- On the source cluster when ``commit`` is received.
+You can enable dual write-blocking, which blocks
+writes on both the source and destination clusters. 
```

Comment:
can we delete ", which blocks writes on both the source and destination clusters. " from this sentence in favor of the explanations below?